molnsutil/molnsutil.py

```diff
@@ -69,6 +72,7 @@ class MolnsUtilStorageException(Exception):
 
 import os
 def get_s3config():
```

Comment:
Not that it is so important, but we could change the name to "get_persisistent_storage_config"